src/index.html

```diff
@@ -7,8 +7,30 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img class="logo" src="images/logo.png"
+        alt="Moyo" >
+      </a></img>
```

Comment:
```suggestion
      <a href="#home">
        <img 
          class="logo" 
          src="images/logo.png"
          alt="Moyo" 
        >
      </a>
```
Fix formatting. Also, remove the unnecessary closing </img> tag, as <img> elements are self-closing.
![image](https://github.com/mate-academy/layout_moyo-header/assets/96409563/f24c9dce-ca87-4f93-97d6-0b12d9bf000b)